qa/standalone/osd/osd-rep-recov-eio.sh

```diff
@@ -119,12 +119,24 @@ function rados_get_data() {
     COUNT=$(ceph pg $pgid query | jq '.info.stats.stat_sum.num_objects_repaired')
     test "$COUNT" = "1" || return 1
 
+    local object_osds=($(get_osds $poolname $objname))
+    local primary=${object_osds[0]}
+    local bad_peer=${object_osds[1]}
     inject_$inject rep data $poolname $objname $dir 0 || return 1
     inject_$inject rep data $poolname $objname $dir 1 || return 1
+    # Force primary to pull from the bad peer, so we can repair it too!
+    set_config osd $primary osd_debug_feed_pullee $bad_peer || return 1
+    rados_get $dir $poolname $objname || return 1
+
+    # Wait until automatic repair of bad peer is done
+    wait_for_clean || return 1
+
+    inject_$inject rep data $poolname $objname $dir 0 || return 1
+    inject_$inject rep data $poolname $objname $dir 2 || return 1
     rados_get $dir $poolname $objname || return 1
```

Comment:
@dzafman Tests updated. Without the fix, this rados_get command will hang since all three copies are broken now.

Reply:
@xiexingguo It is supposed to hang and does the rados_get with hang option that does a timeout and succeeds if it does timeout.

rados_get $dir $poolname $objname hang || return 1

Reply:
Nope. I mean without https://github.com/ceph/ceph/pull/26806/commits/f598c4f58e4da6643d27245a77c24f567ef1af75, the newly added test case will fail, because we won't automatically repair bad peers on pulling error before.

The patch together with the fixer works fine now!